kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -4344,13 +4336,22 @@ def _get_detail(self):
                             raise kvmagent.BlockJobError(
                                 "the block job status is abnormal, details is ioHung. Please check backup storage and backup network.")
 
+                        processed = total - remain
+                        result = jsonobject.JsonObject()
+                        result.put("processed", processed)
+                        result.put("total", total)
+                        result.put("stage", "migrating")
+                        result.put("unit", "bytes")
+                        if total > 0:
+                            result.put("percent", min(100, int(round(processed * 100.0 / total))))
```

Ruff (0.15.5)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)